src/Raven.Server/Documents/ChangesClientConnection.cs

```diff
@@ -876,6 +905,14 @@ public void HandleCommand(string command, string commandParameter, BlittableJson
             {
                 WatchTopology();
             }
+            else if (Match(command, "watch-aggressive-caching"))
+            {
+                _aggressiveChanges = true;
```

Comment:
sure that we do not need to use counter like in `_watchAllIndexes` ?

Reply:
The scenario we have here is that the _entire_ connection is dedicated just for aggressive caching.